src/components/UserDivs.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import UserDetail from "./UserDetail";
+
+class UserDivs extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+    };
+  }
+  handleUserClick(event) {
+    const users = this.props.props;
+    console.log("user", event, users);
+    this.setState({
+      idOfUser: this.id,
+    });
+  }
+  render() {
+    const users = this.props.props;
+    console.log("i got props", users);
+    const userD = users.map((user, index) => {
```

Comment:
Always try to use descriptive variable names. Rather make them a little bit longer than too short...